### PR TITLE
unit tests for internal/cleanup 

### LINKED ISSUE #241: cleanup: add unit tests

### PR DESCRIPTION
**Pull request by jan25**
Partially Fixes #241

Add basic tests for cleanup handler creator funcs.

Left a TODO: to add unit test for `export.database.DeleteFilesBefore`

### CODE REVIEW

File: internal/cleanup/cleanup_test.go [hunk 1]
```diff
@@ -15,10 +15,106 @@
...
 	"testing"
 	"time"
+
+	"github.com/google/exposure-notifications-server/internal/database"
+	"github.com/google/exposure-notifications-server/internal/serverenv"
+	"github.com/google/exposure-notifications-server/internal/storage"
 )
 
+func TestNewExposureHandler(t *testing.T) {
+	emptyDB := &database.DB{}
```
Comment (sethvargo) [inline internal/cleanup/cleanup_test.go:29]: Use testDB

File: internal/cleanup/cleanup_test.go [hunk 2]
```diff
@@ -15,10 +15,106 @@
...
+	ctx := context.TODO()
```
Comment (sethvargo) [inline internal/cleanup/cleanup_test.go:30]: Use context.Background instead

File: internal/cleanup/cleanup_test.go [hunk 3]
```diff
@@ -15,10 +15,106 @@
...
+				} else if handler.env != tc.env {
+					t.Fatalf("got %+v: want %v", handler.env, tc.env)
+				}
+			}
+		})
+	}
+}
+
+func TestNewExportHandler(t *testing.T) {
+	emptyStorage := &storage.GoogleCloudStorage{}
```
Comment (sethvargo) [inline internal/cleanup/cleanup_test.go:71]: Use the initializers for these please

**Comment by jan25** [comment]
Thanks for the review! Is CI failing at new tests?

Also, for testDB implementation, i noticed locally - this line seems to look one directory above root of this project, is that intended?

**Comment by sethvargo** [comment]
Current failures (sorry, working on making this public): 

```text
2020/05/24 17:13:31 creating test DB: open /home/prow/go/src/github.com/google/migrations: no such file or directory
FAIL	github.com/google/exposure-notifications-server/internal/cleanup	0.298s
ok  	github.com/google/exposure-notifications-server/internal/database	0.011s	coverage: 17.4% of statements
ok  	github.com/google/exposure-notifications-server/internal/envconfig	0.056s	coverage: 80.0% of statements
ok  	github.com/google/exposure-notifications-server/internal/export	0.040s	coverage: 13.8% of statements
```

We've moved some things around in recently, and I think it may have broken the database tests.

**Comment by sethvargo** [comment]
@jan25 you may want to hold off until we get #428 merged.

File: internal/export/database/export_test.go [hunk 4]
```diff
@@ -588,3 +588,5 @@ func TestAddExportFileSkipsDuplicates(t *testing.T) {
 		t.Fatalf("bucket name mismatch got %q, want %q", got.BucketName, wantBucketName)
 	}
 }
+
+// TODO TestDeleteFilesBefore
```
Comment (mikehelmick) [inline internal/export/database/export_test.go:592]: this should either reference an issue or a user

TODO(#427)
or
TODO(username)

or - just revert this file if the note is just for yourself.

Comment (jan25) [reply internal/export/database/export_test.go]: I mentioned my username. I think TODO can be resolved in separate PR though it is related to same issue.

**Comment by sethvargo** [comment]
#428 is merged if you wanna rebase and follow the same pattern for the testDB as in there.

**Comment by jan25** [comment]
I already rebased and updated this PR. Could you review?

File: internal/cleanup/cleanup_test.go [hunk 5]
```diff
@@ -15,10 +15,106 @@
...
+	}{
+		{
+			name: "nil Database",
+			env:  serverenv.New(ctx),
+			err:  fmt.Errorf("missing database in server environment"),
+		},
+		{
+			name: "Fully Specified",
+			env:  serverenv.New(ctx, serverenv.WithDatabase(testDB)),
+			err:  nil,
```
Comment (sethvargo) [inline internal/cleanup/cleanup_test.go:45]: Nit: default value for error is `nil`, so you could omit this.

**Pull request merged**